src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static final int MAX_NUMBER = 101;
```

Comment:
Avoid using static methods or fields in your classes as per the checklist. Instead, you should create an instance of the class where you need to use it.